Xamarin.Forms.Platform.iOS/Renderers/ShellFlyoutContentRenderer.cs

```diff
@@ -18,16 +18,23 @@ public class ShellFlyoutContentRenderer : UIViewController, IShellFlyoutContentR
 
 		public ShellFlyoutContentRenderer(IShellContext context)
 		{
+			_shellContext = context;
+
 			var header = ((IShellController)context.Shell).FlyoutHeader;
 			if (header != null)
 				_headerView = new UIContainerView(((IShellController)context.Shell).FlyoutHeader);
-			_tableViewController = new ShellTableViewController(context, _headerView, OnElementSelected);
+
+			_tableViewController = CreateShellTableViewController();
```

Comment:
Is there any concern with calling a virtual method from the constructor?

Reply:
Yea best practices dictate that I should move this :-) 

I think moving this to a lazy instantiation + wiring this up inside an override of *LoadView* would be the better way to put this together

Reply:
Sounds like a much better plan